src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/DividerSamplePage.xaml

```diff
@@ -0,0 +1,110 @@
+﻿<Page x:Class="Uno.Material.Samples.Content.Controls.DividerSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:controls="using:Uno.Material.Controls"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  mc:Ignorable="d">
+
+	<Page.Resources>
+		<DataTemplate x:Key="DefaultItemTemplate">
+			<Grid MinWidth="200">
+				<Grid.ColumnDefinitions>
+					<ColumnDefinition Width="*" />
+					<ColumnDefinition Width="auto" />
+				</Grid.ColumnDefinitions>
+
+				<TextBlock Text="Item"
+						   Foreground="Black" />
+
+				<SymbolIcon Symbol="Favorite"
+							Foreground="Black"
+							Grid.Column="1" />
+			</Grid>
+		</DataTemplate>
+		
+		<DataTemplate x:Key="CustomItemTemplate">
+			<Grid MinWidth="200">
+				<Grid.ColumnDefinitions>
+					<ColumnDefinition Width="*" />
+					<ColumnDefinition Width="auto" />
+				</Grid.ColumnDefinitions>
+
+				<TextBlock Text="Item"
+						   Foreground="White" />
+
+				<SymbolIcon Symbol="Favorite"
+							Foreground="White"
+							Grid.Column="1" />
+			</Grid>
+		</DataTemplate>
+	</Page.Resources>
+
+	<ScrollViewer Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<StackPanel>
+
+			<smtx:XamlDisplay UniqueKey="Divider_Default">
+				<StackPanel Background="{StaticResource MaterialSurfaceBrush}"
+							Margin="8">
+					<ContentPresenter ContentTemplate="{StaticResource DefaultItemTemplate}" />
```

Comment:
the text doesnt look good in dark mode:
![image](https://user-images.githubusercontent.com/2359550/91077532-3a959180-e60f-11ea-9bb4-f4071b04e451.png)
just use textblock + body typo style, or set a theme foreground